presets/alpine-packages.json5

```diff
@@ -5,7 +5,7 @@
     {
       // Update pinned alpine packages in Dockerfile.
       "customType": "regex",
-      "fileMatch": [ "Dockerfile" ],
+      "managerFilePatterns": ["/Dockerfile/"],
```

Comment:
This one is borderline harmful.

The fact that the renovate documentation doesn't bother to mention whether regular expressions are anchored to lines or not just makes this hard to figure out, but I'm going to guess they aren't (because the docs suggest to use `^foo` and `bar$`, which wouldn't make sense if they were anchored by default)...

... which means that this matches anything where Dockerfile is a substring: `Dockerfile.foo`, `foo.Dockerfile`, `not-a-Dockerfile` and other stuff like that. Either this is intentional, and it should be documented, or it isn't, and it should be correctly anchored.

Or you use the new shiny 🤮 feature and leave it as `Dockerfile` (glob, not regular expression), which would match a file named Dockerfile and nothing else.